Source/engine/point.hpp

```diff
@@ -45,6 +45,13 @@ struct Point {
 		return *this;
 	}
 
+	constexpr Point &operator-=(const Size &size)

```

Comment:
If the `Displacement(Size)` constructor was not explicit these operators could be removed and let the conversion to `Displacement` allow `Point -= Size` to use `Point::operator-=(Displacement)` instead.

Reply:
IIRC @glebm was the one who suggested the explicit constructor for this.

I honestly don't mind either way. Please discuss and let me know what you guys agree on

Reply:
The explicit form has already been decided on.